client/src/components/EmailTemp.js

```diff
@@ -1,7 +1,195 @@
+
+const templateOptions = [
+    {
+        key: 'SW',
+        text: 'Social Worker',
+        value: '1',
+        //Seong's image here!!
+        image: { avatar: true, src: '.jpg' },
+    },
+    {
+        key: 'Doc',
+        text: 'Doctor',
+        value: '2',
+        //Seong's image here!!
+        image: { avatar: true, src: '.jpg' },
+    }
+]
+
 
 class EmailTemp extends React.Component {
+
+
+
+
+    constructor (props) {
+        super(props);
+        //calling the template options
+
+        this.state = {
+            name: 'name',
+            lName: 'lName',
+            val: 'val',
+            city: 'city',
+            //illness: 'illness',
+            phone: 'phone'
+        };
+        this.handleChange = this.handleChange.bind(this);
+        this.handleChangeLast = this.handleChangeLast.bind(this);
+        this.handleChangeValue = this.handleChangeValue.bind(this);
+        this.handleChangeCity = this.handleChangeCity.bind(this);
+        // this.handleChangeIllness = this.handleChangeIllness.bind(this);
+        this.handleChangePhone = this.handleChangePhone.bind(this);
+    }
+
+    handleChange(e) {
+        this.setState({
+            name: e.target.value
+        });
+    }
+
+    handleChangeLast(e) {
+        this.setState({
+            lName: e.target.value
+        });
+    }
+
+    handleChangeCity(e) {
+        this.setState({
+            city: e.target.value
+        });
+    }
+
+
+    //Possible for later implementation
+    // handleChangeIllness(e) {
+    //     this.setState({
+    //         illness: e.target.illness
+    //     });
+    // }
+
+    handleChangePhone(e) {
+        this.setState({
+            phone: e.target.value
+        });
+    }
+
+    handleChangeValue = (e, {value}) => {
+        this.setState({
+            val: value
+        })
+    }
+
+    template (value) {
+        if(value == 1) {
+            return (
+                <Container text>
+                    <Header as='h2'>Your Email To Your Social Worker:</Header>
+                <p>
+                    Hello. My name is {this.state.name} {this.state.lName}. I
+                    wanted to reach out to you for assistance. I currently live
+                    in a food desert - I usually stay in the {this.state.city}
+                    area. I have chronic illnesses,
+                    and take medication that requires me to have regular meals.
+                    Given the fact that I live in a food desert, I would like
+                    to find methods/places where I can get food on a regular
+                    basis. I would love to speak with you about any
+                    opportunities you think would be helpful for me.
+                    You can contact me by phone at {this.state.phone}. <br/>
+                    Thank you, <br/>
+                    {this.state.name} {this.state.lName}
+                </p>
+                </Container>
+            );
+        } else if (value == 2) {
```

Comment:
In JavaScript, use `===` instead of `==` for comparisons.  `===` will also require that the types of the variables on both sides matches, so for instance, `2 == "2"` is true but we might not want that, whereas `2 === "2"` is false.